app/models/manager_refresh/inventory_collection.rb

```diff
@@ -146,6 +149,11 @@ def object_index_with_keys(keys, object)
       keys.map { |attribute| object.public_send(attribute).to_s }.join(stringify_joiner)
     end
 
+    def find_or_build(manager_uuid)
+      # FIXME: splat manager_ref
+      data_index[manager_uuid] || build({ manager_ref.first => manager_uuid })
```

Comment:
ok, let's keep it that way and do this in a follow up, where all the other `manager_uuid` methods in here are also updated.